test/selenium/ftu_test.js

```diff
@@ -1,10 +1,100 @@
-var webdriver = require('selenium-webdriver'),
-  until = require('selenium-webdriver').until;
+var assert = require('assert'),
+  test = require('selenium-webdriver/testing'),
+  webdriver = require('selenium-webdriver');
 
-var driver = new webdriver.Builder().
-  forBrowser('firefox').
-  build();
+test.describe('setup page', function() {
+  var driver;
+  this.timeout(8000);
+  before(function() {
+    driver = new webdriver.Builder().
+      forBrowser('firefox').
+      build();
+  });
+  beforeEach(function() {
+    driver.get('http://foxbox.local:3000/');
+  });
+  after(function() {
+    driver.quit();
+  });
 
-driver.get('http://localhost:3000/');
-driver.wait(until.titleIs('FoxBox'), 10000);
-driver.quit();
+  test.it('should be titled FoxBox', function () {
+    return driver.wait(webdriver.until.titleIs('FoxBox'), 5000).then(function(value) {
```

Comment:
Ah, I had added jshint only for `static/`, added it for `test/selenium/` now as well.